client/src/components/columns.jsx

```diff
@@ -1,55 +1,38 @@
 import React, { Component } from "react";
 import axios from "axios";
-let arrhappy = [];
-let arrsad = [];
-const api = "http://localhost:3001/sites";
 
 class Columns extends Component {
   state = {
-    data: [{ id: 0, name: "", urls: ["", ""] }],
     happy: [],
     sad: [],
     faking: []
   };
   componentDidMount() {
     this.getDataFromDb();
-    this.parseData(this.state.data);
   }
 
   getDataFromDb = () => {
-    axios.get(api).then(result => {
-      console.log(result.data);
-      let variable = [];
-      variable = result.data;
-      this.setState({ data: variable });
+    axios.get("/sites/check").then(result => {
+      this.updateStatus(result.data);
```

Comment:
it's best to try keep minimal state - if it's not required - do not store the whole request